src/filesystem.c

```diff
@@ -263,16 +263,13 @@ int64_t Fs_Load(const char *filename, void **buffer) {
 	} fs_block_t;
 
 	file_t *file;
-	if ((file = Fs_OpenRead(filename))) {
-
-		if (!PHYSFS_setBuffer((PHYSFS_File *) file, FS_FILE_BUFFER)) {
-			Com_Warn("%s: %s\n", filename, Fs_LastError());
-		}
```

Comment:
This is a great find, @Paril. I must have hastily read the docs, and assumed that you must always initialize a buffer. My loading times (which were already quite small) are now down to 2 seconds on the first map, and under 1 second on subsequent maps (with exceptions like Wake).